gulpfile.js

```diff
@@ -7,6 +7,8 @@ var maps = require('gulp-sourcemaps');
 var gulpSequence = require('gulp-sequence');
 
 var files = [
+    './node_modules/regenerator-runtime/runtime.js',
```

Comment:
add this line since getCrossDomainCSSRules is an async function

Reply:
I don't believe this is appropriate. Instead of including this file into the flot distribution, it's preferable to declare the new dependency as a 'dependency' and not a 'devDependency'.